AssemblerLib/VectorMatrixAssembler.h

```diff
@@ -38,6 +38,14 @@ class VectorMatrixAssembler
 
     ~VectorMatrixAssembler() {}
 
+    void setX(GLOBAL_VECTOR_ const * x, GLOBAL_VECTOR_ const * x_prev_ts)
+    {
+        assert((!x == !x_prev_ts) && "either no or both inputs have to be set");
+        assert((!x) || x->size() == x_prev_ts->size());
+        _x = x;
+        _x_prev_ts = x_prev_ts;
+    }
```

Comment:
I use `x_prev_ts` for the assembly of the global equation system. E.g. for a parabolic problem the linear equation system to solve is `M x' + K x = b`, where `x' = (x - y) / dt`
So (currently) in need both `x` and `y := x_prev_ts`.
Maybe later on this can be moved to a different place.